docs/jsdoc/options.jsdoc

```diff
@@ -36,6 +36,10 @@
  * The filename of the template. Required for inclusion and caching unless 
  * you are using {@link module:ejs.renderFile}. Also used for error reporting.
  * 
+ * @property {String}  [root='undefined']
```

Comment:
It is correct. See http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values

> #### An optional parameter and default value
> ```js
> /**
 * @param {string} [somebody=John Doe] - Somebody's name.
 */
function sayHello(somebody) {
    if (!somebody) {
        somebody = 'John Doe';
    }
    alert('Hello ' + somebody);
}
```